api/database/seeders/ClassificationRolesSeeder.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Seeder;
+
+class ClassificationRolesSeeder extends Seeder
```

Comment:
I suggest making this singular to match our other seeders.
```suggestion
class ClassificationRoleSeeder extends Seeder
```